docusaurus.config.js

```diff
@@ -145,7 +152,12 @@ module.exports = {
           ],
         },
       ],
-      copyright: "Copyright © 8x8, Inc.",
+      logo: {
+        alt: "8x8 Footer Logo",
+        src: "https://web-cdn.jitsi.net/meet8x8com_6214.2958/images/8x8-copyright-icon.svg",
+        href: "/",
```

Comment:
Not sure this is a good link destination...

Reply:
Where should I put it? jitsi.org or 8x8.vc?

Reply:
8x8.com